analysis/analyze-hotspots/src/main/java/com/esri/samples/analyze_hotspots/AnalyzeHotspotsSample.java

```diff
@@ -189,9 +190,11 @@ public void start(Stage stage) {
                     }
                   });
                 } else {
-                  new Alert(AlertType.ERROR, "Time range must be within " + minDate.toString() + " and " + maxDate
-                      .toString()).show();
+                  // display alert and reset time range in date pickers
+                  new Alert(AlertType.ERROR, "Time range must be within " + minDate + " and " + maxDate).show();
```

Comment:
```suggestion
                  new Alert(AlertType.ERROR, "Time range must be within " + minDate + "\nand " + maxDate).show();
```

@Rachael-E one more change - add a new line in this error so that the max date shows in the alert:

![image](https://user-images.githubusercontent.com/56556822/137301978-9cf2f7c1-9d2a-43bc-8cfa-11eb15b6434b.png)

to 

![image](https://user-images.githubusercontent.com/56556822/137302093-efc8e47c-368d-4ff3-ba55-0211e95fd411.png)